solidity/contracts/token/libs/MovableCollateralRouter.sol

```diff
@@ -136,35 +138,30 @@ abstract contract MovableCollateralRouter is TokenRouter {
             amount
         );
 
-        uint256 nativeValue = 0;
         // charge the rebalancer any bridging fees denominated in the collateral
         // token to avoid undercollateralization
-        uint256 collateralFee = 0;
-        for (uint256 i = 0; i < quotes.length; i++) {
-            if (quotes[i].token == token()) {
-                collateralFee += quotes[i].amount;
-            }
-            if (quotes[i].token == address(0)) {
-                nativeValue += quotes[i].amount;
-            }
+        uint256 collateralBridgeQuote = quotes.extract(token());
```

Comment:
I think we had mentioned that explaining the logic and intended invariants we enforce here (and dont enforce here) would be good to add?

Reply:
I personally think it would still useful to add more documentation here. I.e. your point here that we want to avoid undercollateralization, but not that the rebalancer pays all the fees is not clear at all